### ISSUE TITLE
Save training loss history

### ISSUE THREAD

**Issue by Gollini** [UW-EXP/GLOBEM#6]
Hello, I would like to save the training loss history to see the progress using different optimizers.
I am not familiar with TensorFlow. How can I extract this information?
Kind Regards.

**Comment by OrsonXu** [comment]
Hi Gollini, you may want to use add another callback function to access the loss from the model. Please check an example here: https://stackoverflow.com/questions/65302030/access-loss-and-model-in-a-custom-callback

You can consider adding a callback to the https://github.com/UW-EXP/GLOBEM/blob/6b7581d476bc1fc1f68b9488b56089563df7b900/algorithm/dl_erm.py#L71
